tests/snapshots_test.rs

```diff
@@ -65,3 +71,91 @@ fn during_merge_conflicts() {
     assert_ne!(status.commit_hash, status.base_hash);
     assert_eq!(status.dura_branch, format!("dura/{}", status.base_hash));
 }
+
+#[test]
+#[serial]
+fn test_commit_signature_using_dura_config() {
+    let tmp = tempfile::tempdir().unwrap();
+    let mut repo = util::git_repo::GitRepo::new(tmp.path().to_path_buf());
+    repo.init();
+    repo.set_config("user.name", "git-author");
+    repo.set_config("user.email", "git@someemail.com");
+
+    env::set_var("DURA_CONFIG_HOME", tmp.path().to_path_buf());
```

Comment:
If you used `util::Dura`, it has a way to deal with environment variables without setting them globally. The trouble is, it needs to subprocess to work, so you'd have to change this to use `dura capture`.

Reply:
I couldn't find a nice way to get the last commit hash from `dura capture`, so I resorted into this which is simpler and easier to follow. Will see if I can find a way to do it nicely.